grpc-api/src/test/java/io/deephaven/grpc_api/runner/DeephavenApiServerSingleAuthenticatedBase.java

```diff
@@ -0,0 +1,51 @@
+package io.deephaven.grpc_api.runner;
+
+import io.deephaven.grpc_api.DeephavenChannel;
+import io.deephaven.proto.backplane.grpc.HandshakeRequest;
+import io.deephaven.proto.backplane.grpc.HandshakeResponse;
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.ClientInterceptors;
+import io.grpc.ForwardingClientCall;
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import io.grpc.MethodDescriptor;
+import org.junit.Before;
+
+import java.util.UUID;
+
+public abstract class DeephavenApiServerSingleAuthenticatedBase extends DeephavenApiServerTestBase {
+
+    DeephavenChannel channel;
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        final DeephavenChannel channel = channel();
+        final HandshakeResponse result =
+                channel.sessionBlocking().newSession(HandshakeRequest.newBuilder().setAuthProtocol(1).build());
+        // Note: this one-shot authentication / handshake should be enough for the duration of unit tests.
```

Comment:
No need to "hope", let's crank up the timeout in the TestBase:
```java
                .withSessionTokenExpireTmMs(300000) // defaults to 5 min
```

We can just upgrade that to a MAX_INT. I have hit this 5m timeout plenty of times during long debug sessions, and I would simply bump the number up on the process I was attaching to.